tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/FundingSourceControllerTests.cs

```diff
@@ -18,17 +16,86 @@
 namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
 {
     [TestFixture]
-    public sealed class FundingSourceControllerTests
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class FundingSourceControllerTests
     {
         [Test]
-        public void Constructor_NullOrderRepository_ThrowsException()
+        public void Constructor_NullOrderRepository_ThrowsArgumentNullException()
         {
             var builder = FundingSourceControllerBuilder.Create()
                 .WithOrderRepository(null);
 
             Assert.Throws<ArgumentNullException>(() => builder.Build());
         }
 
+        [Test]
+        public async Task GetAsync_OrderDoesNotExist_ReturnsNotFound()
+        {
+            const string orderId = "DOESNOTEXIST";
+            var context = FundingSourceControllerTestContext.Setup();
+            context.Order = null;
+
+            using var controller = context.Controller;
```

Comment:
Do you have to extract out controller, why don't you just use it in the next line

Reply:
I think it was a pattern used elsewhere, doesn't have to but will lose the `using` keyword. I will change them.